extensions/memory-operands-1/README.md

```diff
@@ -99,6 +99,9 @@ as though the operand values were the size specified by the instruction's Operan
 
 # Added Instruction
 
+These instruction formats apply to any instruction which encodes operands using an `ABM` byte _unless_ that instruction already performs a memory
```

Comment:
This is still duplicated, it is already mentioned above at the start of "Added instruction formats". MO1 and MO2 are inconsistent in that regard.

Reply:
🤦
Should be fixed now...